descriptors/ModuleDescriptor-template.json

```diff
@@ -739,6 +749,16 @@
       "displayName": "Circulation - use circulation rules to get all matching request policies",
       "description": "Apply circulation rules to get all matching request policies"
     },
+    {
+      "permissionName": "circulation.rules.notice-policy.get",
```

Comment:
The endpoints (`/circulation/rules/patron-notice-policy` and `/circulation/rules/patron-notice-policy-all`?) relating to these policies don't seem to be described, please can they be added.

Reply:
Is this a different description than lines 754 - 761?

I'm not sure what's missing or where.

Reply:
Apologies, the endpoints for determining patron notice policies need defining for Okapi to use (and these permissions binding to them).

Similarly to https://github.com/folio-org/mod-circulation/blob/master/descriptors/ModuleDescriptor-template.json#L371-L394

Let me know if that doesn't make sense.

Reply:
Ok, thanks. I've updated the handlers with the descriptions of the notice-policy endpoints.